app/controllers/series_controller.rb

```diff
@@ -213,6 +213,18 @@ def update_notes
     render :partial => 'investigation_sort.html'
   end
 
+  def stale
+    stale_ds = DataSource.where('last_run_in_seconds < ?', Time.now.days_ago(2).to_i).order(:last_run_in_seconds => :asc)
```

Comment:
I kinda thought that might be an issue, but then I thought: maybe it's ok because it's a relatively rarely used page, and for admins rather than end users. So... is the solution then to write SQL by hand to return the series names and ids?